### DISCUSSION TITLE
64Drive - Unfinished feature support possible?

### DISCUSSION THREAD

**Discussion by Drakonas** [Polprzewodnikowy/N64FlashcartMenu#148] [Q&A] [answered]
Is there a remote chance that support for hardware in the 64Drive could be realized that was never finished? The 64Drive has a WiFi ESP8266 802.11b/g/n chip inside but support was never completed for it.

I am just unsure if this project is only aiming to change the frontend, and if the actual hardware functions are mostly off-limits or being ignored for the project.

I understand either. I am simply asking for information and maybe striking up conversation to see if there is a possibility this could be on a roadmap some time in the future (given someone knows how)?

**Comment by Polprzewodnikowy** [comment, marked as answer]
AFAIK the WiFi feature was never finished in the 64drive's firmware, so it's not possible to access it at all. Even if there was some way to communicate with the ESP8266 chip, nobody put the work to reverse engineer the protocol and test it.

**Reply by Drakonas** [reply]
No worries. I wasn't sure what the extent was of the project. I understand. Thank you!
